safe_control_gym/envs/benchmark_env.py

```diff
@@ -386,6 +386,7 @@ def before_step(self, action):
         # Sanity check (reset at least once).
         self._check_initial_reset()
         # Save the raw input action.
+        action = np.array([action]).flatten()
```

Comment:
Do we want to forbid non-flat actions spaces?
Wouldn't a warning be more appropriate?

Reply:
I found this error b/c some controllers return floats for the cartpole rather than a numpy array, which causes an error downstream. im guessing a controller could reasonably also send a 2D array. this extra line makes sure the action from this point forward is a 1D array

Reply:
I support making `action` a NumPy array, but I'd replace the flattening with a check on `action.ndim` that raises an exception so that the developer of a new controller can decide whether that was intentional or not

Reply:
Hmm but then we'd need a few if statements, something like this: 

`if type == float or int -> np.array([action])`
`if type == list -> np.array(action)`
`if action.ndim != 1 -> raise Exception`

How does that look?

Reply:
admittedly idk why it would be an int lol

Reply:
I think you could do `np.array([action]).squeeze()` instead of the first 2 lines, no?

Reply:
Yeah you right, I'll fix that now